Dashboard/app/js/lib/controllers/general-controllers-common.js

```diff
@@ -8,19 +10,23 @@ FLOW.dashboardLanguageControl = Ember.Object.create({
             { label: "Português", value: "pt" },
             { label: "Tiếng Việt", value: "vi"}],
 
-  languageChanged: function () {
-    var localeUrl = '/ui-strings.js?locale=' + this.dashboardLanguage;
-    $.ajax({
-      url: localeUrl,
-      complete: function () {
-        location.reload(false);
-      },
-    });
-  }.observes('dashboardLanguage'),
+  languageChanged: function(){ 
+     var localeUrl = '/ui-strings.js?locale=' + this.dashboardLanguage;
+     $.ajax({
+       url: localeUrl,
+       complete: function () {
+         location.reload(false);
+       },
+     });
+     
+  },
 });
- 
- 
-FLOW.selectedControl = Ember.Controller.create({
+
+FLOW.selectedControl = Ember.Controller.create(observe({
+  'this.selectedSurveyGroup': 'deselectSurveyGroupChildren',
+  'this.selectedSurvey': 'deselectSurveyChildren',
+  // 'this.selectedSurveyOPTIONQuestions': 'OptionQuestions',
```

Comment:
Do we still need this one? if not maybe better to remove completely.

Reply:
it was commented out before so I just left it that way